Update _asset.html.erb

shilovk opened the pull request:
I [change][1] class Pictury to class Ckeditor::Epicture < Ckeditor::Asset
and from editor picture browser (.../ckeditor/pictures?CKEditor=memory_about&CKEditorFuncNum=1&langCode=ru) was an error:

NoMethodError in Ckeditor/pictures#index Showing /usr/local/rvm/gems/ruby-1.9.3-p545/gems/ckeditor-4.1.0/app/views/ckeditor/shared/_asset.html.erb where line #4 raised: undefined method 'epicture_path' for #<#<Class:0x00000005a652e8>:0x0000000799b158> Extracted source (around line #4): 1: <%= content_tag(:div, :id => dom_id(asset), :class => "gal-item", :"data-url" => image_path(asset.url_content)) do %> 2: <%= link_to image_tag('ckeditor/filebrowser/images/gal_del.png', 3: :title => I18n.t('ckeditor.buttons.delete')), 4: polymorphic_path(asset, :format => :json), 5: :remote => true, 6: :method => :delete, 7: :data => {:confirm => t('ckeditor.confirm_delete')}

For correct work in _asset.html.erb I change:

polymorphic_path(asse, :format => :json) to image_path(asset.url_content)

And works it fine.
Please fix in gem. Thanks

[1]: http://stackoverflow.com/a/31070828/3563993

kyledecot commented:
I'm not sure but I feel this would break other parts of the gem. Using `image_path` here assumes that the asset is always going to be a picture. This partial could be rendered w/ a `CKEditor::Asset` and be a lot of different things.

Why did you feel the need to rename the model in the first place? The namespacing should prevent it from colliding w/ things in your app

shilovk commented:
First:
I rename the model to `epicture.rb`, because before my project have also model `picture.rb` and it bring the error. I think other people have also [this error][1], because that often in projects already has a model `picture.rb`.

Second:
Do I understand correctly [that][2] `polymorphic_path` not work in Rails 3-4? 

[1]: https://stackoverflow.com/questions/17789559/conflict-between-picture-model-and-ckeditorpicture-model
[2]: http://apidock.com/rails/ActionController/PolymorphicRoutes/polymorphic_path

kyledecot commented:
I've had no problem using `polymorphic_path` in any of my rails 3/4 apps. Perhaps we could try and fix the initial problem of the namespacing instead of sidestepping the issue. What was the initial error that you got?

shilovk commented:
my project use: ruby-1.9.3-p545, rails 3.2.14, mysql2 0.3.16, paperclip 4.2.1
That I doing:
1) for ActiveRecord + paperclip I do: `rails generate ckeditor:install --orm=active_record --backend=paperclip`
2) after a have 2 models `picture.rb` 
![1](https://cloud.githubusercontent.com/assets/1587824/8391138/2c322218-1cbe-11e5-8080-e5e51cf5d366.png)
3) my `picture.rb`:
```
class Picture < ActiveRecord::Base
	belongs_to :post, :foreign_key => "obj_id", :conditions => ['contr = ?', 'post']
```
4) my `post.rb`:
```
class Post < ActiveRecord::Base
  has_many :pictures, :foreign_key => "obj_id", :conditions => ['contr =?', 'post'], :dependent => :destroy
```
5) I open to view post, there is the error `Post::Picture`:
```
NameError in Posts#show

Showing /var/www/mytgymnew-development/mytgymnew/app/views/posts/show.html.haml where line #54 raised:

uninitialized constant Post::Picture
Extracted source (around line #54):

51:     = short_date(@post.created_at) +" "+@post.created_at.year.to_s
52:   .posts_show_full
53:     %p= @post.full.mb_chars.gsub(/\n|\r/, "<p></p>").try(:html_safe)
54:   - if @post.pictures.size > 0
55:     %div{:style => "margin-top:20px;"}
56:       %table
57:         - @post.pictures.each_with_index do |picture, index|
```
6) so I renamed ckeditor model to `epicture.rb`.

kyledecot commented:
I just created a [dummy rails project](https://github.com/kyledecot/ckeditor-namespacing-issue) w/ a top level `Picture` model and I am not encountering this error. I'm not sure what you're particular issue is but the StackOverflow post you linked to already had a much similar answer in the form of specifying the class name on the association. 

In any case I still feel that your change would break other models that used the `_asset` partial when the asset isn't an image.
